### DISCUSSION TITLE
How to view the base-images uploaded to Dagster Cloud UI

### DISCUSSION THREAD

**Discussion by m94jain** [dagster-io/dagster#18815] [Q&A] [answered]
Hi,

I am trying to see the existing base images uploaded to Dagster Cloud in the UI. Is there a way to do this?

For context - I have successfully finished the steps below:

1. Uploaded a base-image to my Dagster Cloud instance using the command:
```
$ dagster-cloud serverless upload-base-image local-image:tag
```

2. Deployed my code location using the uploaded base image using the command:
```
dagster-cloud serverless deploy-python-executable \
  --location-name <location_name> \
  --package-name <package_name> \
  --base-image-tag sha256_518ad2f92b078c63c60e89f0310f13f19d3a1c7ea9e1976d67d59fcb7040d0d6
```

I uploaded a few base images on Dagster Cloud while trying to make the steps above work. I am now trying to clean the redundant base images but am having trouble finding a view showing all the base images uploaded to Dagster Cloud.

**Comment by prha** [comment, marked as answer]
We don't have a view in Dagster Cloud that displays the uploaded images.

Aside from clean-up, is there a use-case for this view you're particularly interested in?  Are you running into any issues currently from not cleaning the redundant images?

**Comment by m94jain** [comment]
Thanks for letting me know, @prha. As of now, clean-up is the only use case I have, and I am not running into any issues from not cleaning the redundant images.
